apps/src/javalab/Javalab.js

```diff
@@ -100,13 +111,15 @@ Javalab.prototype.init = function(config) {
         MOBILE_PORTRAIT_WIDTH
       );
     }
+    config.afterInject && config.afterInject();
```

Comment:
style nit: can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here (charlie enabled this in our babel config recently)

Reply:
✔️